include/s3select_oper.h

```diff
@@ -782,80 +773,68 @@ class value
 
     throw base_s3select_exception("operands not of the same type(numeric , string), while comparision");
   }
-  bool operator<=(const value& v)
+
+  bool operator<=(const value& v) const
   {
-    if ((is_null() || v.is_null()) || (is_nan() || v.is_nan())) {
+    if (is_null() || v.is_null() || is_nan() || v.is_nan()) {
       return false;
-    } else {
-      return !(*this>v);
-    } 
+    }
+
+    return !(*this > v);
   }
   
-  bool operator>=(const value& v)
+  bool operator>=(const value& v) const
   {
-    if ((is_null() || v.is_null()) || (is_nan() || v.is_nan())) {
+    if (is_null() || v.is_null() || is_nan() || v.is_nan()) {
       return false;
-    } else {
-      return !(*this<v);
-    } 
+    }
+
+    return !(*this < v);
   }
   
-  bool operator!=(const value& v)
+  bool operator!=(const value& v) const
   {
-    if ((is_null() || v.is_null()) || (is_nan() || v.is_nan())) {
+    if (is_null() || v.is_null() || is_nan() || v.is_nan()) {
       return true;
-    } else {
-      return !(*this == v);
-      } 
+    }
+
+    return !(*this == v);
   }
-  
-  template<typename binop> //conversion rules for arithmetical binary operations
-  value& compute(value& l, const value& r) //left should be this, it contain the result
+
+  template <typename binop>  // conversion rules for arithmetical binary operations
+  value& compute(value& l, const value& r)  // left should be this, it contain the result
   {
     binop __op;
 
-    if (l.is_string() || r.is_string())
-    {
+    if (l.is_string() || r.is_string()) {
```

Comment:
pleas keep the original indentation scheme (for consistency and ease of review):
```
if () 
{
  // code
}
```

Reply:
on previous PR , the following utility was used , it keeps consistency.

astyle --style=stroustrup --style=break --align-pointer=type --align-reference=type --pad-comma --add-braces --indent=spaces=2

Reply:
@galsalomon66 this is great. also, like the idea of making these changes in a dedicated PR. 
lets not mix indentation/formatting changes with other code changes

Reply:
no problem. Will undo.
I was using clang-format (which is the de-facto standard tool) with "my" specific configuration.

Reply:
ok, is it possible to share clang-format and the specific configuration?